src/Effect/Monad.agda

```diff
@@ -58,7 +58,16 @@ record RawMonad (F : Set f → Set g) : Set (suc f ⊔ g) where
   unless : Bool → F ⊤ → F ⊤
   unless = when ∘′ not
 
+-- When level g=f, a join/μ operator is definable
+
+module Join {F : Set f → Set f} ⦃ M : RawMonad F ⦄ where
```

Comment:
I'm a little bit hesitant about making this an instance argument. As the rest of the PR shows, the current monad hierarchy isn't exactly setup for using instances. 

If we are going to introduce instances, I'd rather have a concrete plan in mind for the whole hierarchy?

Could we make this explicit instead?

Reply:
Ah, I see... and I certainly do not have a concrete plan for `instance`s for the `Effect` hierarchy (and in any case, that was *not* the point of this PR). Will proceed as suggested.